vendor/calcite-1_20_0/build.gradle

```diff
@@ -21,10 +21,10 @@ plugins { id 'org.apache.beam.vendor-java' }
 description = "Apache Beam :: Vendored Dependencies :: Calcite 1.20.0"
 
 group = "org.apache.beam"
-version = "0.1"
+version = "0.2"
 
 def calcite_version = "1.20.0"
```

Comment:
Is it possible to bump the calcite_version as well?

Reply:
sure, I will bump the calcite_version to `1.21.0`